_layouts/default.html

```diff
@@ -41,6 +43,8 @@
     {{ content }}
 
     {% if site.service_worker != false %}{% include site-sw.html %}{% endif %}
+
+    {% if site.include_before_end_js %}{% include site-before-end.html %}{% endif %}
```

Comment:
Same as the other include, remove the if statement and remove the variables options from the config